operator/src/main/java/org/bf2/operator/managers/OperandOverrideManager.java

```diff
@@ -98,6 +99,43 @@ public Map<String, Object> getBrokerConfig() {
         public void setBrokerConfig(Map<String, Object> kafkaConfig) {
             this.brokerConfig = kafkaConfig;
         }
+
+        public Map<String, ListenerOverride> getListeners() {
+            return listeners;
+        }
+
+        public void setListeners(Map<String, ListenerOverride> listeners) {
+            this.listeners = listeners == null ? Map.of() : listeners;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class ListenerOverride {
```

Comment:
I wonder if a lot of this will get more concise with records, once this project moves to Java 17+